api/src/controllers/authController.ts

```diff
@@ -65,34 +67,51 @@ export default async (fastify: FastifyInstance) => {
     url: '/login',
     schema: fastify.getSchema(ESchemasIds.POSTLogin),
     handler: async function (
-      request: Request<{ Body: { pseudo?: string, mail?: string, password: string } }>,
+      request: Request<{
+        Body: { pseudo?: string; mail?: string; password: string };
+      }>,
       reply: Reply
     ) {
       const { _errorService, _authService } = this;
       const { mail, pseudo, password } = request.body;
 
-      const privateUser = await _authService.getPrivateUser(pseudo ? { pseudo } : { mail });
-      if (!privateUser) // Check if user exists
+      const privateUser = await _authService.getPrivateUser(
+        pseudo ? { pseudo } : { mail }
+      );
+      if (!privateUser)
+        // Check if user exists
         _errorService.send(EErrorMessages.INVALID_USER, 404);
       // Check if password match
-      const isPasswordCorrect = await _authService.verifyPassword(privateUser.id, password);
+      const isPasswordCorrect = await _authService.verifyPassword(
+        privateUser.id,
+        password
+      );
```

Comment:
```suggestion
      const isPasswordCorrect = await _authService.verifyPassword(privateUser.id, password);
```
4 lines for that is to much